constellations/orion/src/controllers/transaction_controller.go

```diff
@@ -84,3 +85,8 @@ func DeleteTransaction(c *gin.Context) {
 		c.Status(http.StatusOK)
 	}
 }
+
+func GetAllPaymentTypes(c *gin.Context) {
+	paymentTypes := []string{domains.PAY_PAYPAL, domains.PAY_CASH, domains.PAY_CHECK, domains.CHARGE, domains.REFUND}
+	c.JSON(http.StatusOK, paymentTypes)
```

Comment:
if the list is defined in transaction.go,
You can then just refer to the list constant here:
c.JSON(http.StatusOK, domains.ALL_TRANSACTION_TYPES)